ocaml/xapi-consts/api_errors.ml

```diff
@@ -493,6 +493,7 @@ let pool_auth_enable_failed_duplicate_hostname = pool_auth_enable_failed^"_DUPLI
 let pool_auth_disable_failed = pool_auth_prefix^auth_disable_failed
 let pool_auth_disable_failed_wrong_credentials = pool_auth_disable_failed^auth_suffix_wrong_credentials
 let pool_auth_disable_failed_permission_denied = pool_auth_disable_failed^auth_suffix_permission_denied
+let pool_auth_disable_failed_invalid_account = pool_auth_disable_failed^auth_suffix_invalid_account
```

Comment:
The error actually can be found also in https://github.com/xapi-project/xen-api/commit/7abecc42eb44ed3e00fdf1648c9d588e0a86d618